src/pipeline_blocks.py

```diff
@@ -538,22 +565,22 @@ def stacking_normalization(features, config, train_mode, suffix, **kwargs):
         features, features_valid = features
 
     normalizer = Step(name='stacking_normalizer{}'.format(suffix),
-                      transformer=SklearnTransformer(Normalizer()),
+                      transformer=SklearnClassifier(Normalizer()),
```

Comment:
It's not a classifier, it's a transformer-and it can be also imported from toolkit. Moreover, in this version this code won't work, because classifier output is 'predicted' instead of 'transformed'.